### PR TITLE
[Debug] Add `resolve-env` option documentation of `debug:config` command

### LINKED ISSUE #17028: [FrameworkBundle] Add `resolve-env` option to debug:config command
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#46821
| PR author(s) | @alexandre-daubois
| Merged in    | 6.2

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Resolves #17028

### CODE REVIEW

File: reference/configuration/debug.rst [hunk 1]
```diff
@@ -16,6 +16,12 @@ key in your application configuration.
     # displays the actual config values used by your application
     $ php bin/console debug:config debug
 
+.. note::
+
+    The ``--resolve-env`` option is available with the ``debug:config``
```
Comment (94noni) [inline reference/configuration/debug.rst:21]: You should(must?) add a version added directive for that new option

**Review by 94noni — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Alexandre, thanks a lot for contributing this feature and its docs.

However, while merging this we reworded it to match the style used in other parts of the docs. We now show the option in an example and we also added the `versionadded` directive as Antoine suggested: https://github.com/symfony/symfony-docs/commit/edd706f22ca69c1d955bbd581db3dd808e67c9b4

File: reference/configuration/debug.rst (hunk 1 above)
Comment (javiereguiluz) [reply reference/configuration/debug.rst:21]: Done while merging. Thanks.

**Comment by alexandre-daubois** [comment]
Great, thanks!

**Pull request merged**